.github/sandbox_workflows.md

```
@@ -0,0 +1,202 @@
+# Documentation for GitHub Actions Pipelines: Sandbox Creation and Sandbox Deletion
+
+This documentation provides an overview of two GitHub Actions workflows used for managing AWS CodePipeline executions: Sandbox Creation and Sandbox Deletion. It includes instructions on the required secrets and how to add them to your GitHub repository.
```

Markdownlint (0.35.0)

Expected: 100; Actual: 253
Line length

(MD013, line-length)